bcda/main.go

```diff
@@ -379,3 +390,46 @@ func getEnvInt(varName string, defaultVal int) int {
 	}
 	return defaultVal
 }
+
+func removeExpired(hrThreshold int) {
+	log.Info("Cleaning up expired jobs...")
+	db := database.GetGORMDbConnection()
+	defer db.Close()
+
+	var jobs []models.Job
+	err := db.Find(&jobs, "status = ?", "Completed").Error
+	if err != nil {
+		log.Error(err)
+	}
+
+	expDir := os.Getenv("FHIR_EXPIRED_DIR")
+	if _, err = os.Stat(expDir); os.IsNotExist(err) {
+		err = os.MkdirAll(expDir, os.ModePerm)
+		if err != nil {
+			log.Error(err)
+		}
+	}
+
+	for _, j := range jobs {
+		t := j.CreatedAt
+		elapsed := time.Since(t).Hours()
+		if int(elapsed) >= hrThreshold {
+
+			id := int(j.ID)
+			jobDir := fmt.Sprintf("%s/%d", os.Getenv("FHIR_PAYLOAD_DIR"), id)
+			expDir = fmt.Sprintf("%s/%d", os.Getenv("FHIR_EXPIRED_DIR"), id)
```

Comment:
Are these actually directory names or rather file names?

Reply:
Good question! They're directory names, the files themselves will be in the job directory and they get moved to the new expDir location. folder --> to folder.